pgmpy/estimators/BayesianEstimator.py

```diff
@@ -0,0 +1,151 @@
+# coding:utf-8
+
+from pgmpy.estimators import BaseEstimator
+from pgmpy.factors import TabularCPD
+from pgmpy.models import BayesianModel
+import numpy as np
+import pandas as pd
+
+
+class BayesianEstimator(BaseEstimator):
+    def __init__(self, model, data, **kwargs):
+        """
+        Class used to compute parameters for a model using Bayesian Parameter Estimation.
+        See `MaximumLikelihoodEstimator` for constructor parameters.
```

Comment:
@chrisittner Please complete this docstring. Add `Parameters` and `Examples` section.